nhlpy/api/edge.py

```diff
@@ -0,0 +1,443 @@
+from typing import Dict, Any
+
+from nhlpy.http_client import HttpClient, Endpoint
+
+
+class Edge:
+    def __init__(self, http_client: HttpClient) -> None:
+        self.client = http_client
+
+    # ========================
+    # SKATER ENDPOINTS
+    # ========================
+
+    def skater_detail(self, player_id: int, season: int = None, game_type: int = None) -> Dict[str, Any]:
```

Comment:
```suggestion
    def skater_detail(self, player_id: str, season: str = None, game_type: int = None) -> Dict[str, Any]:
```

Let's use strings for `player_id` and `season` for this and the other methods, I found it makes using the library a bit easier.  This would match things like stats.player_game_log `player_game_log(self, player_id: str, season_id: str, game_type: int)`